testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/DomainControllerMigrationTestCase.java

```diff
@@ -226,15 +218,7 @@ public void testDCFailover() throws Exception {
 
         hostUtils[2].executeForResult(changeMasterOp);
 
-        restartOp = new ModelNode();
-        restartOp.get(ModelDescriptionConstants.ADDRESS).add(ModelDescriptionConstants.HOST, HOSTS[2]);
-        restartOp.get(ModelDescriptionConstants.OP).set("reload");
-        restartOp.get(ModelDescriptionConstants.RESTART_SERVERS).set(false);
-
-        log.info("Reloading failover-h3 to register to new domain controller.");
-        hostUtils[2].executeAwaitConnectionClosed(restartOp);
-
-        waitUntilHostControllerReady(hostUtils[2]);
+        hostUtils[2].reload(HOSTS[2], null, false, false);
```

Comment:
waitUntilHostControllerReady also waits for the servers, so this should be `hostUtils[2].reload(HOSTS[2], null, false, true);`

Probably, since we are not restarting the servers here we do not need to wait for them, but, let's try to not add a change here. There are other calls in the test suite that were also waiting for the servers even if they are not restarted. If you prefer to change it, please change it on the others too. otherwise, better if you leave it as it was.